src/components/Animator/README.md

```diff
@@ -11,6 +11,12 @@
 | className | string | false | - | The class will be added as a **direct parent class** for all the children it will have|
 | debug | string | false | enter, entering, exit, exiting | Emulates the real animation classes and styles without the need to animate. Just exit your element visible and add to debug different stages. enter: 'enter', entering: 'enter enter-active', exit: 'exit', exiting: 'exit exit-active' |
 | dataHook | string | - | - | will convert to data-hook |
+| onEnter | function | - | - | Callback onEnter  |
+| onEntering | function | - | - | Callback onEntering  |
+| onEntered | function | - | - | Callback onEntered  |
+| onExit | function | - | - | Callback onExit  |
+| onExiting | function | - | - | Callback onExiting  |
+| onExitedAnimation | function | - | - | Callback onExitedAnimation  |
```

Comment:
I think it is enough to have `onExited`, we don't need Animation postfix, to keep it consistent with other props (onEnter, onEntered, etc..)

Reply:
I have changed all names